.github/workflows/test-ci.yml

```diff
@@ -53,6 +53,7 @@ jobs:
         shell: bash
         run: |
           sudo apt install -y libnfft3-dev
+          python -m pip install numpy<2.0.0
```

Comment:
shouldnt we update in pyproject.toml? Like is it a test issue or lack of compatibility?

Reply:
It's more complicated than that, some backend requires numpy<2.0, other don't care

Reply:
Maybe we document the problem in issue and tag it here? That way we know what needs a fix.. Is the pipeline broken currently?